src/libraries/System.Private.CoreLib/src/Microsoft/Win32/SafeHandles/SafeFileHandle.ThreadPoolValueTaskSource.cs

```diff
@@ -33,12 +33,10 @@ private void TryToReuse(ThreadPoolValueTaskSource source)
         /// </summary>
         internal sealed class ThreadPoolValueTaskSource : IThreadPoolWorkItem, IValueTaskSource<int>, IValueTaskSource<long>
         {
-            // Whether to put the operation on a local ThreadPool queue.
-            private const bool PreferLocal = true;
-
             private readonly SafeFileHandle _fileHandle;
             private ManualResetValueTaskSourceCore<long> _source;
             private Operation _operation = Operation.None;
+            private ExecutionContext? _context;
```

Comment:
This should also be null'd out when you clean out state.

Reply:
Done.